crates/pallet-executor/src/lib.rs

```diff
@@ -346,6 +363,29 @@ impl<T: Config> Pallet<T> {
         Ok(())
     }
 
+    fn check_bundle(
+        SignedOpaqueBundle {
+            opaque_bundle,
+            signature,
+            signer,
+        }: &SignedOpaqueBundle,
+    ) -> Result<(), Error<T>> {
+        let msg = opaque_bundle.hash();
+        if !signer.verify(&msg, signature) {
```

Comment:
This looks nicer to me (there are a few places like this):
```suggestion
        if !signer.verify(&opaque_bundle.hash(), signature) {
```

Reply:
Updated.